module-bluetooth/Bluetooth/interface/profiles/HFP/HFP.cpp

```diff
@@ -276,7 +276,7 @@ namespace bluetooth
             scoHandle = HCI_CON_HANDLE_INVALID;
             isAudioRouted                = false;
             isAudioConnectionEstablished = false;
-            sendAudioEvent(audio::EventType::BlutoothHFPDeviceState, audio::Event::DeviceState::Disconnected);
+            audioDevice.reset();
```

Comment:
It is needed otherwise this **https://appnroll.atlassian.net/browse/EGD-7299** bug occures.

Reply:
ok, and why does it crash exactly? right me if I'm wrong but it's not like calling perfectly normal action should crash system.